docs/docs/adrs/adr-017-allowing-inactive-validators.md

```diff
@@ -62,15 +62,105 @@ The following changes to the state are required:
 * Store the provider consensus validator set in the provider module state under the `LastProviderConsensusValsPrefix` key. This is the last set of validators that the provider sent to the consensus engine. This is needed to compute the ValUpdates to send to the consensus engine (by diffing the current set with this last sent set).
 * Increase the `MaxValidators` parameter of the staking module to the desired size of the potential validator
 set of consumer chains.
+* Introduce extra per-consumer-chain parameters: 
```

Comment:
Line 61 above where it writes:
> Introduce the `MaxProviderConsensusValidators` parameter to the provider module, which is the number of validators that the provider module will send to consumer chains.

Shouldn't it be "number of validators that the provider module sends to the consensus engine of the provider."?

Reply:
Yes, thanks for the catch!